--- entice/configure.in~	2005-05-07 23:11:45.000000000 +0000
+++ entice/configure.in	2005-07-02 13:37:00.000000000 +0000
@@ -124,13 +124,7 @@
 AC_SUBST(edje_cflags)
 AC_SUBST(edje_libs)
 
-AC_PATH_PROG(GIMP_REMOTE_BIN, "gimp-remote-2.2", "", $PATH)
-if [ test -z $GIMP_REMOTE_BIN ]; then
-  AC_PATH_PROG(GIMP_REMOTE_BIN, "gimp-remote-2.0", "", $PATH)
-  if [ test -z $GIMP_REMOTE_BIN ]; then
-    AC_PATH_PROG(GIMP_REMOTE_BIN, "gimp-remote", "", $PATH)
-  fi
-fi
+GIMP_REMOTE_BIN=/usr/bin/gimp-remote
 if [ test -z $GIMP_REMOTE_BIN ]; then
   echo "The GIMP remote is not in your \$PATH. Please ensure it is.";
   echo "Read the manual page for you shell as to how to extend your path.";
